.changes/i18n-windows.md

```diff
@@ -0,0 +1,5 @@
+---
+"wry": patch
+---
+
+Fix i18n on windows
```

Comment:
Set webview2 language to match the OS language. This makes i18n functions like `new Date().toLocaleStrin()` behave correctly.